doc/compiler/jitserver/Build.md

```diff
@@ -0,0 +1,144 @@
+<!--
+Copyright (c) 2018, 2019 IBM Corp. and others
+
+This program and the accompanying materials are made available under
+the terms of the Eclipse Public License 2.0 which accompanies this
+distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+or the Apache License, Version 2.0 which accompanies this distribution and
+is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+This Source Code may also be made available under the following
+Secondary Licenses when the conditions for such availability set
+forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+General Public License, version 2 with the GNU Classpath
+Exception [1] and GNU General Public License, version 2 with the
+OpenJDK Assembly Exception [2].
+
+[1] https://www.gnu.org/software/classpath/license.html
+[2] http://openjdk.java.net/legal/assembly-exception.html
+
+SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+
+There are currently 3 supported build procedures. Select the one that best suits your needs.
+
+- [Building the whole VM](#vm). The complete build process (Most correct and recommended). Produces a complete JVM as output (including the JIT).
+- [Building only the JIT](#jit). Choose this if you have an existing SDK and have no need to modify the VM or JCL. This only builds OMR and everything in the `runtime/compiler` directory. Output artifact is `libj9jit29.so`. (Fastest way to get JITServer built)
+- [Building in Docker](#docker). Choose this if you don't want to deal with build dependencies. Takes the longest.
+
+For building the whole VM and building only the JIT, you will have to go through the [Prepare your system](#prerequisites) step.
+For building in Docker, you can skip [Prepare your system](#prerequisites).
+
+If you run into issues during this process, please document any solutions here.
+
+## PREREQUISITES
+- The following example is for Ubuntu18:
+
+```
+apt-get update \
+ && apt-get install -qq -y --no-install-recommends \
```

Comment:
When I tried this on a fresh ubuntu VM, it complained about `libfontconfig1-dev` not installed. And I see the Dockerfile also installs it. So I think we would need it here as well.